src/swap/SwapAmountInput.tsx

```diff
@@ -147,6 +153,28 @@ const SwapAmountInput = ({
           )}
         </Touchable>
       </View>
+      <View testID="SwapAmountInput/FiatValue">
+        <Text style={[styles.fiatValue, { opacity: loading || !swapAmount || !token ? 0 : 1 }]}>
+          {token?.priceUsd && swapAmount?.gt(0) && localCurrencyExchangeRate
+            ? `≈${localCurrencySymbol}${swapAmount
+                .multipliedBy(token.priceUsd)
+                .multipliedBy(localCurrencyExchangeRate)
+                .toFormat(2, BigNumber.ROUND_DOWN)}`
```

Comment:
I think this format should be translatable. Also maybe a space after the approximate symbol?
Why don't we use our `TokenAmount` component here too?
Otherwise we risk deviating from our "standard" way of showing amounts.

Reply:
oh yeah....i was inconsistent about this, i used TokenDisplay in some places in the details section and not here. i've updated to use TokenDisplay, thanks for the catch!